packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1096,6 +1096,20 @@ This section contains some general information about claims, as well as detailed
 * [ID Token](#id-token)
 * [Refresh Token](#refresh-token)
 
+### Token lifetime
+
+When you are using the [Okta Authorization Server](/docs/concepts/auth-servers/#org-authorization-server), the lifetime of the JWT tokens is hard-coded to the following values:
+
+* **ID token:** 60 minutes
+* **Access token:** 60 minutes
+* **Refresh token:** 90 days
+
+When you are using a [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), you can configure the lifetime of the JWT tokens:
+
+* **ID token:** At least five minutes, no more than 60 minutes
+* **Access tokens:** At least five minutes, no more than 24 hours (configurable using an [Access Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-create-access-policies)). See the [Authorization Servers API](/docs/reference/api/authorization-servers/#actions-object).
```

Comment:
The help link does not work. Also do we need to link both prod and api doc here?

Reply:
The help link should work now - they were working on it from the prod side this morning. Which would you rather have? The link to how to configure the timeframe for the token in the access policy or the authorization server api information?